### PR TITLE
Update global-shortcut.md

### PR DESCRIPTION
**Pull request by hankbao**
Make it clear that the `global-shortcut` module should not be used until the ready event of app module gets emitted.

### CODE REVIEW

File: docs/api/global-shortcut.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 The `global-shortcut` module can register/unregister a global keyboard shortcut
 in operating system, so that you can customize the operations for various shortcuts.
 Note that the shortcut is global, even if the app does not get focused, it will still work.
+You should not use this module until the ready event of app module gets emitted.
```
Comment (hokein) [inline docs/api/global-shortcut.md]: Could you please also update the sample code in this doc?

**Comment by hankbao** [comment]
@hokein Sample code updated.

File: docs/api/global-shortcut.md [hunk 2]
```diff
@@ -3,25 +3,33 @@
...
+var app = require('app');
 var globalShortcut = require('global-shortcut');
 
-// Register a 'ctrl+x' shortcut listener.
-var ret = globalShortcut.register('ctrl+x', function() { console.log('ctrl+x is pressed'); })
+app.on('ready', function() {
+    // Register a 'ctrl+x' shortcut listener.
+    var ret = globalShortcut.register('ctrl+x', function() {
+        console.log('ctrl+x is pressed');
+    })
```
Comment (hokein) [inline docs/api/global-shortcut.md]: The code indention should be 2 spaces.

Comment (hankbao) [inline docs/api/global-shortcut.md]: I'll fix it later.

**Comment by hankbao** [comment]
Sample code indention problem fixed.

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**